schemas/contacts-schemas.js

```diff
@@ -0,0 +1,11 @@
+import Joi from 'joi';  // бібліотека, що перевіряє тіло запиту
+
+// Створюємо Joi-схему для post/put-запитів (вимоги до кожного поля)
+const addSchema = Joi.object({
+    name: Joi.string().required(),
+    email: Joi.string().required(),
```

Comment:
You should validate email and phone properly

Reply:
I've done validation.

Reply:
Thank you for the approval. Will you merge my pull request or should I do this?